src/account/rpc.ts

```diff
@@ -3,6 +3,16 @@ import { METHODS } from '../utils/aepp-wallet-communication/schema'
 import { NotImplementedError } from '../utils/errors'
 import { EncodedData } from '../utils/encoder'
 
+// TODO: Remove networkId and use type Parameters<AccountBase['signTransaction']>[1]
+// networkId required to maintain backward compatablity with wallets using SDK v11.0.1 and below
+type RpcSignTransaction = Parameters<AccountBase['signTransaction']>[1] & {
```

Comment:
`signTransaction` already accepts `networkId` and I'm not planning to change it, so this type is not necessary

Reply:
I added the tag `deprecated` to the RpcSignTransaction  filed `networkId` to inform the developers. The main `signTransaction` still accepts the `networkId` but we have removed it from the RPC call. I believe it is important to mark the field `networkId` as deprecated

Reply:
Oh, I see, You have marked it deprecated another way. I hope that raises a warning while developers trying to use the field.